low_level_platform/api/low_level_platform.h

```diff
@@ -19,7 +19,22 @@ extern "C" {
 
 #include "tag.h"
 #include <assert.h>
-#include "lf_atomic.h"
+#include "platform/lf_atomic.h"
+
+// The following two definitions are duplicated from tag.h to break a dependency
+// cycle. TODO: decide whether this (very small) duplication is the right
```

Comment:
Another open TODO. The solution looks fine, but the fact that there is a dependency cycle is an indication that something is off. Perhaps you can document the precise cycle in an issue, and we think about how to resolve it later on?

Reply:
Actually this is a mistake on my part, `tag.h` is in fact already `#include`d above. So this is being deleted. It is still true that `tag.c` is dependent on `low_level_platform.h`, so in this sense there is a cycle, but this problem (if it is truly a problem) can be resolved later.